gui/wxpython/vdigit/toolbars.py

```diff
@@ -144,89 +144,93 @@ def _toolbarData(self):
         self.icons = {
             "addPoint": MetaIcon(
                 img="point-create",
-                label=_("Digitize new point"),
+                label=_("Digitize new point, Ctrl+P"),
```

Comment:
Sorry, I should have been clearer, I believe using parenthesis would look better:

Digitize new point (Ctrl+P)

Reply:
Ok.